lib/middleware/schemas/templates.schema.json

```diff
@@ -127,74 +239,100 @@
           ]
         },
         "properties": {
-          "type": "$defs/elementProperties"
+          "type": "object",
+          "properties": {
+            "titleEN": {
+              "description": "Input label in English",
+              "type": "integer"
```

Comment:
This should probably be a `string`, right?

Reply:
Good catch, thank you!